### ISSUE TITLE
New lines at the end of updates

### ISSUE THREAD

**Issue by Saben65** [cmangos/tbc-db#168]
Ran into issues installing using the auto updater (probably unrelated) so then I tried manually updating and ran into further issues. Numerous commits in the updates folder are missing a new line at the bottom which causes issues if you merge the updates into one file using copy *.sql updates.sql and manually updating.

Are all the updates going to be merged into the full db soon (656 updates are a lot!) or is this something worth spending time fixing?

**Comment by Saltgurka** [comment]
Hopefully I got all of them with this commit: https://github.com/cmangos/tbc-db/commit/ab063dda02bda5f164b37566b1f03106bcde0956

We are working on exporting the last few files from vengeance atm. Once that is done it might be time to finally merge all those updates into the Full_DB file.

Let me know if you have any further issues installing :)

**Comment by Saltgurka** [comment]
On second look there are a ton more files missing newlines than I thought. I will add them, but I may not get to it tonight

**Comment by Saben65** [comment]
I just added a new line to the end of all the files using powershell

```
Add-Content -Path "*.sql" -Value "`r`n"
```

**Comment by Saltgurka** [comment]
This should be all of them now: https://github.com/cmangos/tbc-db/commit/75a9226c6301f7a0ee6c8cfe0eadadd4b657bc48
